CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
 # Changelog
 
+## Version 1.4.0 (2018-10-12)
+
+- BREAKING CHANGE: Add an option `minDragStartDistance` on `Draggable` to define the minimum distance in
```

Comment:
Could this be done without a breaking change by making clickSuppression set minDragStartDistance?